versions.txt

```diff
@@ -1,12 +1,12 @@
 # Represents the latest stable release of the CloudWatch Agent
-cloudwatch-agent=1.300037.0b583
+cloudwatch-agent=1.300040.0
```

Comment:
It this suppose to have the hash appened?

Reply:
can use `1.300040.0b650`

Reply:
Updated